pkgs/development/tools/misc/patchelf/default.nix

```diff
@@ -15,6 +15,18 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
+
+  patches =
+    # These patches fix a MIPS-specific bug in patchelf; we want Hydra
+    # to generate a bootstrap-files tarball for MIPS that includes
+    # this fix.  The patches below can be dropped on the next version bump.
+    lib.optionals stdenv.targetPlatform.isMips [
+      # https://github.com/NixOS/patchelf/pull/380/commits/820da7be8d1e1a49c4831dcb3800ed3b9f11e8a6
+      ./patches/820da7be8d1e1a49c4831dcb3800ed3b9f11e8a6.patch
+      # https://github.com/NixOS/patchelf/pull/380/commits/8db45c6a0c1a4dbbd492ac7fb59c1bca9460fe3e
+      ./patches/8db45c6a0c1a4dbbd492ac7fb59c1bca9460fe3e.patch
```

Comment:
These should just be `fetchpatch`ed instead.

You can use https://github.com/NixOS/patchelf/pull/380.patch for a single combined diff.